system/BaseSpec.cfc

```diff
@@ -516,7 +516,7 @@ component {
 		}
 
 		// Attach this spec to the incoming context array of specs
-		arrayAppend( this.$suitesReverseLookup[ this.$suiteContext ].specs, spec );
+		arrayAppend( variables.this.$suitesReverseLookup[ variables.this.$suiteContext ].specs, spec );
```

Comment:
This is actually incorrect since we REALLY want to put this into the `this` scope.  If not, we are creating them in the `variables` scope.

Reply:
Lucee's implicit variable debugging says it's resolving it from the variables scope? 

C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 503 | variables | THIS | 932
-- | -- | -- | -- | --
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 519 | variables | THIS | 1864
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 527 | variables | THIS | 932

Reply:
Well, then it is wrong, because the `this` scope is completely different than the `variables` scope. I would report this to Micha.